view/stache/doc/helpers/is.md

```diff
@@ -0,0 +1,27 @@
+@function can.stache.helpers.is {{#is expr1 expr2 expr3}}
+@parent can.stache.htags 12
+
+@signature `{{#is expr1 expr2}}BLOCK{{/is}}`
```

Comment:
On pages like http://canjs.com/docs/can.stache.helpers.helper.html we use "..." to signal unlimited number of arguments.

Please change the signature to:

`{{#is expr...}}BLOCK{{/is}}`